src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -104,7 +104,7 @@
 
       <ItemGroup>
         <_sourceToExclude Include="@(Compile)" Condition="$([System.String]::Copy('%(Compile.Identity)').ToLower().EndsWith('assemblyinfo.cs'))" />
-        <_sourceToPackage Include="@(Compile->'%(FullPath)')" PackagePath="contentFiles/cs/netstandard2.0/System/Text/Json/%(FileName)%(Extension)" Exclude="@(_sourceToExclude)"/>
+        <_sourceToPackage Include="@(Compile->'%(FullPath)')" PackagePath="contentFiles/cs/netstandard2.0/Microsoft.Bcl.Json.Sources/%(FileName)%(Extension)" Exclude="@(_sourceToExclude)"/>
```

Comment:
Can we store the `Microsoft.Bcl.Json.Sources` package name in a common location and use that so we only ever have to update it in one place?

Reply:
Sounds good, I'll do that. I will also do that for the path where the sources are dropped so that we can drop the README.md file on the same location.